Gadget.Notifications/Controllers/NotificationsController.cs

```diff
@@ -26,6 +28,15 @@ public NotificationsController(INotificationsService notificationsService)
             return Created("", "");
         }
 
+        [HttpGet("types")]
+        public async Task<IActionResult> GetNotifierTypes()
+        {
+            var canToken = new CancellationToken();
+           await _emailService.SendEmailMessage(new Domain.ValueObjects.EmailMessage("test message", "marcin_mojski@interia.pl"), canToken);
```

Comment:
to jest do usuniecia? bo chcemy tylko typy notyfikacji tutaj zwracac

Reply:
tak tylko sprawdzałem czy chodzi odpalając link z przeglądarki.

Reply:
no ok, ale skoro mergujemy to do mastera to mozna sprzatnac

Reply:
no tak planowałem :D, już leci comit